### PR TITLE
FIX: Address some reliability issues of the functional masking workflow

### PR DESCRIPTION
**Pull request by oesteban**
Addresses nipreps/fmriprep#2731

### CODE REVIEW

File: niworkflows/func/util.py [hunk 1]
```diff
@@ -403,16 +391,8 @@ def init_enhance_and_skullstrip_bold_wf(
...
-            operation="max",
-            kernel_shape="sphere",
-            kernel_size=6.0,
-            internal_datatype="char",
-        ),
-        name="skullstrip_first_dilate",
-    )
-    bet_mask = pe.Node(fsl.ApplyMask(), name="skullstrip_first_mask")
+    bet_dilate = pe.Node(BinaryDilation(radius=6), name="skullstrip_first_dilate")
+    bet_mask = pe.Node(ApplyMask(), name="skullstrip_first_mask")
```
Comment (mgxd) [inline niworkflows/func/util.py:395]: we should probably take the chance to rename these, since we're changing the interfaces altogether

**Review by mgxd — APPROVED** [review]
I think this looks good (at least from locally testing ds210) - would you mind rebasing to `maint/1.4.x` since we'll need to backport this fix to fMRIPrep 21.0.x

**Comment by oesteban** [comment]
Okay, rebased. I'm going to generate a new branch `masks/` so that the mask tests are triggered.

BTW - any ideas of why get data is failing on circle?

**Comment by effigies** [comment]
Just from a quick look, it seems this violates our bug-fix rules. Does it make sense to target 1.5.x instead and just go in 22.0.0?

**Comment by effigies** [comment]
> BTW - any ideas of why get data is failing on circle?

https://github.com/datalad/datalad/pull/6631#issuecomment-1102682597

**Pull request closed without merging**